p2p/transport/webrtc/internal/transport.go

```diff
@@ -0,0 +1,23 @@
+package internal
+
+import (
+	"crypto/rand"
+)
+
+const (
+	uFragAlphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"
+	uFragPrefix   = "libp2p+webrtc+v1/"
+	uFragIdLength = 32
+	uFragIdOffset = len(uFragPrefix)
+	uFragLength   = uFragIdOffset + uFragIdLength
+)
+
+func GenUfrag(n int) string {
```

Comment:
The `n` is unused. Also consider moving this function to the webrtc package?

Reply:
I could, but as I asked previously, I'm not sure on why it would be better either way? I guess you like the code to be close together?

Reply:
Done.